README.md

```diff
@@ -127,12 +127,55 @@ declaration order (i.e `age` first, `person` second). If the first argument to t
 --inputs = "[[42,1722,7], (26, 200), 7432533831]"
 ```
 
+If you're passing in the `bytes` Solidity type as an argument, use the python syntax, for example:
+```bash
+--inputs = "[[10,20], b"\x01\x02"]"
```

Comment:
I don't think it's goint to work. Bash doesn't support nested quotes. The inner quotes must be single, while outer — double, or vice versa